src/editor/app.js

```diff
@@ -397,6 +397,25 @@ UiDriver.registerEventHandler("C_CMD_SET_THEME", function(msg, data, prevReturn)
     editor.setOption("theme", data.name);
 });
 
+UiDriver.registerEventHandler("C_CMD_SET_FONT", function(msg, data, prevReturn) {
+
+    var fontSize = (data.size != "" && data.size>0) ? ("font-size:" + data.size + "px;") : "";
+
+    var fontFamily = data.family != "" ? ("font-family:'" + data.family + "';") : "";
```

Comment:
`data.family` should be escaped in case it contains quotes.
Replacing `'` with `\'` should be enough.

And just to be on the safe side, at line 400 add a `+` before data.size, like this:

    ... ("font-size:" + (+data.size) + "px;") ...